js/app.js

```diff
@@ -0,0 +1,212 @@
+// 1. boolean
+// 2. string
+// 3. Objects
+// 4. Array
+// 5. Objects - with strings
+// 6. Objects - with a strings and an array
```

Comment:
**2/6** for this section: instructions asked you to actually create small examples (oversight?)